src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onSelectTodo: (todo: Todo) => void,
+  selectedTodo: Todo | null,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  onSelectTodo,
+  selectedTodo,
+}) => {
+  const isTodoSelected = todo.id === selectedTodo?.id;
+  const selectTodo = () => (
```

Comment:
We create function names with the verb `handle` or `on`

 ```suggestion
  const onClickSelectTodo = () => (
```